src/website/shared/models/cve.py

```diff
@@ -368,3 +369,41 @@ class AdvisorySeverity(models.TextChoices):
         CRITICAL = "CRITICAL", _("critical")
 
     issues = models.ManyToManyField(NixpkgsIssue)
+
+
+class NixDerivationAffectedView(models.Model):
```

Comment:
Not sure I understand here, shouldn't the list of derivations affected derive from an NixpkgsIssue?

Reply:
that's how the function view `affected_derivation_per_channel_view` is defined.

In any case, I'll push a change soon where I'll get rid of the view table and achieve the same with a queryset.